controllers/httpchaos_controller.go

```diff
@@ -42,6 +44,13 @@ type HTTPChaosReconciler struct {
 func (r *HTTPChaosReconciler) Reconcile(req ctrl.Request) (result ctrl.Result, err error) {
 	logger := r.Log.WithValues("reconciler", "httpfaultchaos")
 
+	if !common.ControllerCfg.ClusterScoped && req.Namespace != common.ControllerCfg.TargetNamespace {
+		// NOOP
+		logger.Info("ignore chaos which belongs to an unexpected namespace within namespace scoped mode",
+			"chaosName", req.Name, "expectedNamespace", common.ControllerCfg.TargetNamespace, "actualNamespace", req.Namespace)
+		return
```

Comment:
Maybe a little bit clearer
```suggestion
		return ctrl.Result{}, nil
```

Reply:
Agree :D